### PR TITLE
984 icu dat not loading

### PR DESCRIPTION
**Pull request by enekid**
Context: [CartoCSS properties text-wrap-width/text-wrap-character not working](https://github.com/CartoDB/support/issues/984)

### CODE REVIEW

**Comment by enekid** [comment]
There are 3 alternatives for setting the ICU_DATA env variable.
IMHO Option 1 should be the preferred way, but it sadly fails.
Option 2 is then the proposed alternative

**Comment by dgaubert** [comment]
Nice work, let's go with option 2.

**Comment by oleurud** [comment]
Option 2. In a fast review, I don't understand why the option 1 not works

File: test/support/test_helper.js [hunk 1]
```diff
@@ -21,6 +21,17 @@ global.environment  = require(__dirname + '/../../config/environments/test');
...
+// CartoCSS properties text-wrap-width/text-wrap-character not working
+function setICUEnvVariable() {
+    const glob = require('glob');
+    const path = require('path');
+    let directory = glob.sync(path.join(__dirname, '../..', '/node_modules/mapnik/lib/binding/*/share/mapnik/icu/'));
+    if (directory && directory.length > 0) {
+        process.env.ICU_DATA = directory[0];
+    }
+}
+setICUEnvVariable();
```
[the remark this answers is not in the document]
Comment (enekid) [reply test/support/test_helper.js]: Ok. It smelled a bit!

File: test/acceptance/label-wrap.js [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+        keysToDelete = {};
+    });
+
+    afterEach(function (done) {
+        if (this.testClient) {
+            this.testClient.drain(done);
+        }
+    });
+
+    it.only("Label should be text-wrapped", function (done) {
```
[the remark this answers is not in the document]
Comment (enekid) [reply test/acceptance/label-wrap.js]: Always. I'm having some problem with travis and I didn't want to run all de test suite. Thanks!

**Comment by enekid** [comment]
@dgaubert !

It would be great being able to use ```require('mapnik')```, **but** [mapnik.js](https://github.com/mapnik/node-mapnik/blob/15064d06ea3258888ca1f28be07a22ee97347f5e/lib/mapnik.js#L6) uses ```require('node-pre-gyp').find``` before setting the ```ICU_DATA``` ENV variable, and for some *wtf* reason, this function call messes around with the ICU configuration. It's what happened in option 1:
```javascript
// The best approach to generate the path where the ICU DATA is stored is by calling binary.find()
// Sadly, it fails. It generates the correct path, but mapnik raises "could not create BreakIterator: U_MISSING_RESOURCE_ERROR"
const binary = require('node-pre-gyp');
let binding_path = binary.find(path.resolve(path.join(__dirname, 'node_modules/mapnik/package.json')));
binding_path = path.dirname(binding_path);
// If we hardcode de path, it works. the call to binary.find seems to be the problem here.
// var binding_path = __dirname + '/node_modules/mapnik/lib/binding/node-v48-linux-x64/';
process.env.ICU_DATA = path.join(binding_path, 'share/mapnik/icu/');
```

So we keep with option 2. Far from elegant but effective :)

**Pull request merged**